### DISCUSSION TITLE
Gdshelpers nicely runs in jupyter notebook

### DISCUSSION THREAD

**Discussion by mniehus** [HelgeGehring/gdshelpers#53] [Show and tell]
Gdshelpers nicely runs in jupyter notebook e.g. first example of tutorial
[gds_tutorial.zip](https://github.com/HelgeGehring/gdshelpers/files/7135338/gds_tutorial.zip)

**Comment by HelgeGehring** [comment]
Nice to hear that it's working fine! Thanks for sharing the example :)
